roles/legal/files/privacy/privacy_statement_de.md

```
@@ -70,33 +71,33 @@ Soweit wir von Ihnen personenbezogene Daten verarbeiten, stehen Ihnen als Betrof
-https://www.datenschutz-bayern.de/
+E-Mail: poststelle@datenschutz-bayern.de  
+https://www.datenschutz-bayern.de/  
 
 #### Weitere Informationen
 
-Für nähere Informationen zur Verarbeitung Ihrer Daten und zu Ihren Rechten können Sie uns unter den oben (zu Beginn von A.) genannten Kontaktdaten erreichen.
+Für nähere Informationen zur Verarbeitung Ihrer Daten und zu Ihren Rechten können Sie uns unter den oben genannten Kontaktdaten des Verantwortlichen erreichen.
 
 ## Informationen zum Internetauftritt
 
 #### Technische Umsetzung
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)